helix-core/src/main/java/org/apache/helix/NotificationContext.java

```diff
@@ -227,4 +228,12 @@ public void setPathChanged(String pathChanged) {
   public void setChangeType(HelixConstants.ChangeType changeType) {
     this._changeType = changeType;
   }
+
+  public boolean getIsChildChange() {
+    return _isChildChange;
+  }
+
+  public void setIsChildChange(boolean cc) {
```

Comment:
You may want to change cc to a more meaningful meaning, like childChange.

Reply:
TFTR. Updated.